ui/src/locales/fa.json

```diff
@@ -1,78 +1,78 @@
 {
-    "faq": {
+    "faq": 
+    {
```

Comment:
Why all in a sudden all `{` are moved to the next line? This is not standard JSON formatting, please revert (you should have `"faq": {` as before, and so on